plugins/services/src/js/reducers/__tests__/JSONMultiContainer-test.js

```diff
@@ -0,0 +1,84 @@
+const Batch = require("#SRC/js/structs/Batch");
+const { combineParsers } = require("#SRC/js/utils/ParserUtil");
+const { combineReducers } = require("#SRC/js/utils/ReducerUtil");
+const JSONMultiContainerParser = require("../JSONMultiContainerParser");
+const JSONMultiContainerReducers = require("../JSONMultiContainerReducers");
+
+describe("JSONMultiContainer", function() {
```

Comment:
I do not see a `JSONMultiContainerParser`, so I left the tests in the current file.

As for your comment about going from `Batch` to `JSON` to `Batch` again... I added a test for what I think this means, but I could be wrong. Effectively the test now does this: `JSON` -> `Batch` -> `JSON` -> `Batch` -> `JSON`